stl/src/filesys.cpp

```diff
@@ -267,9 +267,10 @@ _FS_DLL unsigned long long __CLRCALL_PURE_OR_CDECL _File_size(const wchar_t* _Fn
 // 1908 is leap year #2
 // 1968 is leap year #17
 
-#define WIN_TICKS_PER_SECOND 10000000ULL
 
-#define WIN_TICKS_FROM_EPOCH (((1970 - 1601) * 365 + 3 * 24 + 17) * 86400ULL * WIN_TICKS_PER_SECOND)
+constexpr UINT64 _Win_ticks_per_second = 10000000ULL;

```

Comment:
Could this be `uint64_t` instead, to be consistent with the `int64_t`s I see elsewhere in this file? (And on line 272.)